src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    key: 'Nothing was pressed yet',
```

Comment:
```suggestion
    key: null,
```
It is much more flexible